src/fs-plus.coffee

```diff
@@ -95,7 +96,7 @@ fsPlus =
 
   # Public: Returns true if a file or folder at the specified path exists.
   existsSync: (pathToCheck) ->
-    isPathValid(pathToCheck) and (statSyncNoException(pathToCheck) isnt false)
+    isPathValid(pathToCheck) and (isThere.sync(pathToCheck) isnt false)
```

Comment:
Atom Shell natively implements `statSyncNoException` which will be faster since it doesn't spend time capturing the stack trace during exception generation.

So I'm :-1: on this change since it will no longer make use of that optimization if it uses the isThere library.